### ISSUE TITLE
Executing a shell command

### ISSUE THREAD

**Issue by canerkoseren** [spring-cloud/spring-cloud-dataflow#4509]
Hi all,

I am triggering a task over scdf dashboard by using deployer properties. I want to execute a shell command right after the pod creation, before the application initialization. 

I looked at the [spring-cloud-dataflow-docs/configuration-kubernetes](https://docs.spring.io/spring-cloud-dataflow/docs/current/reference/htmlsingle/#configuration-kubernetes), but I could not find a proper property. Is it possible to execute a shell command as I mentioned?

the version of spring-cloud-data-flow 2.5.0.

Thanks :)

**Comment by canerkoseren** [comment]
I migrate the issue under 'spring-cloud-deployer-kubernetes'

https://github.com/spring-cloud/spring-cloud-deployer-kubernetes/issues/439

**Comment by canerkoseren** [comment]
Hi all,

An update: I have upgraded the version to 2.8.0-SNAPSHOT.

With these version, I started to deploy a side-car container by using the property `deployer.appl.kubernetes.additional-containers` .

With these approach, the side-car container should be deployed before the application container. The application container should be started after side-car container is in ready state.

I considered to trigger a task by adding `initContainer` property, but in this case an image which I did not want would be deployed.

How can I sort the deployments of these containers?
